src/dotnet/commands/dotnet-new/Program.cs

```diff
@@ -96,16 +96,24 @@ public static int Run(string[] args)
             app.Description = "Initializes empty project for .NET Platform";
             app.HelpOption("-h|--help");
 
-            var lang = app.Option("-l|--lang <LANGUAGE>", "Language of project [C#|F#]", CommandOptionType.SingleValue);
-            var type = app.Option("-t|--type <TYPE>", "Type of project", CommandOptionType.SingleValue);
+            var csharp = new { Name = "C#", Alias = new[] { "c#", "cs", "csharp" }, TemplatePrefix = "CSharp", Templates = new[] { "Console", "Web", "Lib", "xunittest" } };
+            var fsharp = new { Name = "F#", Alias = new[] { "f#", "fs", "fsharp" }, TemplatePrefix = "FSharp", Templates = new[] { "Console" } };
+
+            var languages = new[] { csharp, fsharp };
+
+            string langValuesString = string.Join(", ", languages.Select(l => l.Name));
+            var typeValues = 
+                from l in languages
+                let values = string.Join(", ", l.Templates)
+                select $"Valid values for {l.Name}: {values}.";
+            string typeValuesString = string.Join(" ", typeValues);
+
+            var lang = app.Option("-l|--lang <LANGUAGE>", $"Language of project    Valid values: {langValuesString}.", CommandOptionType.SingleValue);
+            var type = app.Option("-t|--type <TYPE>", $"Type of project        {typeValuesString}", CommandOptionType.SingleValue);
```

Comment:
I think this is fine as-is for this PR. We need to fix as part of upcoming `new3` work.